NBitcoin/RPC/RPCClient.cs

```diff
@@ -1562,6 +1587,21 @@ public class Bip9SoftFork
 		public List<Bip9SoftFork> Bip9SoftForks { get; set; }
 	}
 
+	public class TransactionInfo

```

Comment:
Shouldn't it be called rather `RawTransactionInfo`? I think `TransactionInfo` is too generic.

Reply:
Sure. I will do it.